modules/apps/oauth2-provider/oauth2-provider-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay OAuth2 Provider API
 Bundle-SymbolicName: com.liferay.oauth2.provider.api
-Bundle-Version: 7.0.1
+Bundle-Version: 8.0.0
```

Comment:
Everything looks great except I'm unclear why the bundle was made to have a MAJOR version update? Everything else seems to suggest a MINOR update should have been sufficient (i.e. `7.1.0`).

Reply:
Glad it meets the requirements! That's very odd about the major bump. I'll investigate